state/saveAndRestore.go

```diff
@@ -574,6 +574,9 @@ func (ss *SaveState) RestoreFactomdState(s *State) { //, d *DBState) {
 	}
 	// Set this, as we know it to be true
 	s.DBHeightAtBoot = ss.DBHeight
+
+	// set DBHeightBase to avoid recursively loading process lists
+	s.ProcessLists.DBHeightBase = ss.DBHeight
 	pl := s.ProcessLists.Get(ss.DBHeight)
```

Comment:
If we added additional change  w/ noted in comments to ProcessList.go - it may be that this should change to
```
	pl := s.ProcessLists.Get(ss.DBHeight+1)
```